lighthouse/helpers/plates.py

```diff
@@ -426,6 +432,15 @@ def update_mlwh_with_cog_uk_ids(samples: List[Dict[str, str]]) -> None:
             db_connection.close()
 
 
+def supplier_name_for_control(dart_row):
+    args = {
+        "control_type": dart_row[FIELD_DART_CONTROL],
+        "source_barcode": dart_row[FIELD_DART_SOURCE_BARCODE],
+        "source_coordinate": dart_row[FIELD_DART_SOURCE_COORDINATE],
+    }
+    return "{control_type} control: {source_barcode}_{source_coordinate}".format(**args)
```

Comment:
What benefit is constructing the supplier name this way rather than a format string like `f"{dart_row[FIELD_DART_CONTROL]} control: {dart_row[FIELD_DART_SOURCE_BARCODE]}_...."`? The current implementation just seems a little overly complicated

Reply:
That is to fit everything in the same line so it passes the lint

Reply:
That could be achieved with backslashes I think, e.g.
```
return "{dart_row[FIELD_DART_CONTROL]} control: " \
"{dart_row[FIELD_DART_SOURCE_BARCODE]}_" \
...
```

Reply:
This as is, so am happy for this not to be changed